### PR TITLE
Assignment4 - task6 and task7

### PR DESCRIPTION
**Pull request by Alfonsogm**

### CODE REVIEW

File: Assignment4/Alfonsogm-160068/task06.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/18Fxo7kncTMtrqISgNnvdjXVl4_977T-r
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/Alfonsogm-160068/task06.py]: won't work in python (this is for the notebooks only). Try and run this script

File: Assignment4/Alfonsogm-160068/task07.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+''',
+  initNs = {"ns": ns,"rdfs":RDFS}
+)
+# Visualize the results
+for r in g.query(q2):
+  print(r.subj)
+print("\n")
+for s, p, o in g.triples((None, None, ns.Person)):
+  print(s)
+
```
Comment (dgarijo) [inline Assignment4/Alfonsogm-160068/task07.py:80]: You are missing listing all inividuals of the subclasses of Person

File: Assignment4/Alfonsogm-160068/task07.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+"""**TASK 7.3: List all individuals of "Person" and all their properties including their class with RDFLib and SPARQL**
+
+"""
+
+q3= prepareQuery('''
+Select ?subj ?p ?obj 
+Where {
+  ?subj ?p ?obj .
+  ?subj ?p1 ns:Person
```
Comment (dgarijo) [inline Assignment4/Alfonsogm-160068/task07.py]: this is listing all the properties linked to Person, not all instances.
Also, you are missing subclasses. Incorrect

File: Assignment4/Alfonsogm-160068/task07.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+''',
+  initNs = {"ns": ns,"rdfs":RDFS}
+)
+# Visualize the results
+for r in g.query(q3):
+  print(r.subj,r.p,r.obj)
+print("\n")
+for s, p, o in g.triples((None, None, ns.Person)):
+  for s1,p1,o1 in g.triples((s,None,None)):
+    print(s1,p1,o1)
```
Comment (dgarijo) [inline Assignment4/Alfonsogm-160068/task07.py]: missing subclasses of person. Almost correct

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please fix the issues

**Comment by Alfonsogm** [comment]
Issues ready to be reviewed.

**Pull request merged**